Documentation/concepts/kubernetes/compatibility-table.rst

```diff
@@ -86,7 +90,10 @@
 +-----------------+----------------+
 | v1.10.4         | 1.23.3         |
 +-----------------+----------------+
+| v1.10.5         | 1.23.3         |
++-----------------+----------------+
 | v1.10           | 1.23.3         |
 +-----------------+----------------+
-| latest / master | 1.24.1         |
+| latest / master | 1.24.2         |
+| 1.24.1          |                |
```

Comment:
this looks odd

Reply:
Fixed. This is a problem because master didn't update the resource versions for the v2alpha1 CRDs.